src/System.Data.SqlClient/src/System/Data/SqlClient/SqlBuffer.NetCoreApp.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
```

Comment:
can you please remove line 5- line 9

Reply:
Done, they were copied from original non-netcore file so perhaps some cleanup elsewhere might be a good idea.